docs/faststore/docs/reference/project-structure/updating-the-cli-package-version.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Updating the '@fastore/cli' package version"
+---
+
+If you need to update the [`@fastore/cli`](https://developers.vtex.com/docs/guides/faststore/project-structure-overview#packagejson) package, follow these instructions.
+
+1. Open your FastStore codebase in the code editor of your preference.
+
+2. Open the terminal and run the following:
+
+    ```bash
+    yarn add @faststore/cli@latest
+    ```
+
+3. The following message will appear:
+
+```bash
+error Running this command will add the dependency to the workspace root rather than the workspace itself, which might not be what you want - if you really meant it, make it explicit by running this command again with the -W flag (or --ignore-workspace-root-check).
+info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
+```
```

Comment:
@Mariana-Caetano I didn't get this message 🤔 How did you got it?

Reply:
@hellofanny I believe I got this message because I ran this command at the root of the project. When I run into a specific from the project, this message doesn't appear.